pkg/hubble/parser/getters/getters.go

```diff
@@ -76,3 +77,10 @@ type LinkGetter interface {
 	// containing the ifindex if the link name cannot be determined.
 	Name(ifIndex uint32) string
 }
+
+// PodMetadataGetter ...
```

Comment:
nit: I know it's the case for most of the getter above in this file, but I think we should either document this properly or drop the comment.

Reply:
This particular getter warranted some description, so I added one. :)